src/components/Relation.vue

```diff
@@ -315,9 +329,12 @@
 
           // Destroy previous table
           if (this.$table) {
-            this.$table.destroy(true);
-            this.$table     = null;
-            this.table.rows = [];
+            /** @TODO remove "data_from_server" and use table.ajax.reload(); instead */
+            this.$table.destroy(!opts.data_from_server); // In case of data_from_server set .destroy(false), https://datatables.net/reference/api/destroy() 
```

Comment:
@volterra79 we will make use [ajax.reload](https://datatables.net/reference/api/ajax.reload) otherwise the input field (search filter) loses keyboard focus if you are too much slow in typing.

Reply:
@Raruto i'll check

Reply:
> we have an infinite loop

Yes I know, the `DataTable` part should be completely rethought, ie. much more similar to:

https://github.com/g3w-suite/g3w-client/blob/fe580a74abe8ba75f266deff5a68398c18ca16a6/src/components/Table.vue#L701-L716

Furthermore, in future we should also consider replacing the `Relation.vue` component with `Table.vue` (more or less they do the same thing).